recipes/unittest2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unittest2" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "22882a0e418c284e1f718a822b3b022944d53d2d908e1690b319a9d3eb2c0579" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - no-argparse-req.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python <3
+    - pip
+    - six
+    - traceback2
+  run:
+    - python <3
+    - six
```

Comment:
Should we add the version constraint to `six`?

Reply:
The only version of `six` that we package are all much larger than what's required, so I tend to lean towards simplicity. No strong opinion though.

Reply:
Personally would be inclined to add it for consistency. Certainly shouldn't cause any harm to be more consistent with the developer's intent. May help some very oddball situations as well.